marimo/_plugins/stateless/image.py

```diff
@@ -89,6 +89,7 @@ def _normalize_image(src: ImageLike) -> Image:
 def image(
```

Comment:
Adding `caption` in the middle of the arguments is a breaking change, because users may be supplying arguments as positional arguments.

`caption` should instead be added as the last argument:

```python
def image(
    src: ImageLike,
    alt: Optional[str] = None,
    width: Optional[int] = None,
    height: Optional[int] = None,
    rounded: bool = False,
    style: Optional[dict[str, Any]] = None,
    caption: Optional[str] = None,
) -> Html
```

Reply:
learned, and fixed, thank you :)

Reply:
thank you!